src/providers/push-notifications/push-notifications.ts

```diff
@@ -70,10 +70,27 @@ export class PushNotificationsProvider {
           config.productsUpdates.enabled
         )
           this.subscribeToTopic('productsupdates');
+
+        setInterval(() => {
```

Comment:
If users decide to turn off notifications, we should kill setInterval when unsubscribe.